src/types/Todo.ts

```diff
@@ -0,0 +1,11 @@
+export interface Todo {
+  id: number;
+  userId: number;
+  title: string;
+  completed: boolean;
+}
+
+export interface PatchedTodo {
```

Comment:
Can we use sth like this? https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys